### PR TITLE
fix(tenable): api model fix

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**
Main issue : [issue](https://github.com/SekoiaLab/integration/issues/1815)

## Summary by Sourcery

Update Tenable asset models to align with current API responses and prevent validation errors.

Bug Fixes:
- Allow AssetInfo aes_score_v3 and acr_score_v3 fields to accept float values from the Tenable API.
- Allow TagsObject.added_by to be null when the Tenable API omits or nulls this field.

Documentation:
- Document the Tenable AssetInfo and TagsObject model fixes in the changelog.

Tests:
- Add regression tests ensuring AssetInfo accepts float and null score fields and defaults them to None when absent.
- Add regression tests ensuring TagsObject.added_by accepts both null and string values, including when used within AssetInfo tags.

Chores:
- Bump Tenable connector manifest version to 1.0.18.

### CODE REVIEW

File: Tenable/tests/asset_connector/test_vulnerability_asset.py [hunk 1]
```diff
@@ -1075,3 +1076,79 @@ def test_get_vulnerabilities_checkpoint_advances_when_all_vulns_have_no_cve(
     assert len(results) == 0
     expected_ts = int(datetime.fromisoformat(ts2).timestamp())
     assert tenable_asset_connector._latest_time == expected_ts
+
+
+# ==================== Regression tests for model validation bug fixes ====================
+
```
Comment (mchupeau-sk) [inline Tenable/tests/asset_connector/test_vulnerability_asset.py]: I think you can remove this line

Comment (TOUFIKIzakarya) [reply Tenable/tests/asset_connector/test_vulnerability_asset.py]: Yes i can do that, and apply also copilot suggestion

**Review by mchupeau-sk — APPROVED** [review]
LGFM
